WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -125,7 +125,6 @@ decisions = {
 					NOT = { culture = ROOT }
 				}
 			}
-			holy_order = no
```

Comment:
If rulers of Holy Orders don't convert to the local culture, you end up with a French Afghanistan or an Assyrian Mongolia, or a Tangut Russia (three examples of Holy Order cultures). This is a severe oversight on Paradox's part that must be rectified.

Unless you have an alternative means to address this, like making it impossible for Holy Order provinces to flip culture, which would be unrealistic.